src/Components/Blazor/Build/testassets/standalone/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace standalone
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```

Comment:
Note that this app isn't meant to run. It's got enough things in here (targeting the 3.0 release) to build and publish. But that's enough to verify that the build targets execute and do linking etc.